### ISSUE TITLE
Is this project deprecated?

### ISSUE THREAD

**Issue by a9udn9u** [connors/photon#135]
The last meaningful code commit was almost a year ago, doesn't seem it's still being actively developed.

**Comment by behrtam** [comment]
Maybe we should create a fork as something like `photon` is needed in the electron world.

**Comment by jarek-foksa** [comment]
Check out the [Xel project](https://github.com/jarek-foksa/xel) which has very similar goals.

**Comment by mikezila** [comment]
I looked into Xel, and it's cool for sure, but it doesn't really compare to the ease of photon.  To make something that looks as decent as even the generic photon starting point from the docs you have a <x-box> waterfall and still need manual styling to make buttons their correct sizes and distances from siblings.

Just trying to recreate a simple media player I made in photon required a massive number of boxes and manual widget placement and sizing.

**Comment by morriswinkler-simple** [comment]
@connors what's your stand to this rumors. Any updates to come?

**Comment by IonicaBizau** [comment]
@connors Any update on this?

**Comment by NemoStein** [comment]
He seems to be pretty active in private repos here on Github... =/

**Comment by DanielRuf** [comment]
Well, that's not great. Generally I recommend moving such big projects to an own org.

**Comment by ElyaConrad** [comment]
@DanielRuf Yes you are correct but @connors , the owner of this repository, does not provide any kind of possibility to add stuff to this project for me. A @NemoStein  already said, it is still pretty active in my personal Repo that extends PhotonKit to a lot of components: https://github.com/MauriceConrad/PhotonKit-Components

But of course this is not the best case. The best case would be a completly new and extensible framework that also supports Windows Metro UI. I'm absolutely interested in doing such a new project but this will need a lot of work. The project has to implement its own way of documentation and modularity. And I am not a windows developer and therefore I need support for windows parts. Mostly in CSS because JavaScript logic would be more general.

But such a project is a big thing. We have to rethink a lot of things to make a clean and modularity solution..

**Comment by ElyaConrad** [comment]
# Photon 1.0 RELEASE

## Dear Photon User!

I am very proud to announce the release of Photon version 1.0! You are a follower of Photon and maybe you already know my extension framework „PhotonKit Components“ that extends the original Photon project on a high level. Sadly the original developer @connors was not abled to accept my contributions or answer me at least one time because he seems to be inactive since more than 2 years. This is very sad because Photon is a great tool to design native looking UI’s and there was a lot to do.
I was the only active developer who still keeps the project up to date.

![ShowReel](https://dev.maurice-conrad.eu/img/photon/demo.png)

Somedays I thought about the future of the project and the last commitment of @connors was 2 years ago. I realised that the only solution for the future of the project would be a complete new version 1.0

This release is a hard fork of the original project that redefines everything hardly and replaces the PhotonKitComponents project I made.

## Getting Started

```bash
npm install electron-photon
```

Just have a look at the „ShowReel“ application or ask me for questions ;-)

[GitHub](https://github.com/MauriceConrad/Photon)

[NPM](https://www.npmjs.com/package/electron-photon)


### Questions? Ideas?

[conr.maur@googlemail.com](mailto:conr.maur@googlemail.com)

**Kind regards,**
<br>**Maurice Conrad**

**Comment by DanielRuf** [comment]
> @connors was not abled to accept my contributions or answer me at least one time because he seems to be inactive since more than 2 years.

At least I see activity in the last days.

Any, great work. Will check it when I find the time.

**Comment by ElyaConrad** [comment]
If this is the case, I would be very happy to work together with @coonors and merge together what we have.

But at the moment the last commit within the original Photon repository was on 24 Feb 2017 . And this was just a docs commitment..


But as I said: If @coonors would do something , I will be very happy ;-)
